scripts/normalize-by-median.py

```diff
@@ -192,6 +192,9 @@ def get_parser():
                         dest='single_output_filename',
                         default='', help='only output a single'
                         ' file with the specified filename')
+    parser.add_argument('--append', default=False, action='store_true',
+                        help='append reads to the outputfile. '
+                        'Only with -o specified')
```

Comment:
I would word the help text this way: "appends reads instead of replacing the output file (only if -o/--out is specified)'